docs/cli-reference.md

```
@@ -92,11 +94,35 @@ Writes the first-run evidence pack to `.diagram` by default:
 - `architecture.mmd`
 - `report.html`
 
-When `--base` or `--head` is supplied and refs resolve, scan also writes
+### `archscope agent [path]`
+
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)